### ISSUE TITLE
Administrator can change existing user details

### ISSUE THREAD

**Issue by janinakimtrohlev** [buerokratt/Buerokratt-Chatbot#126]
AS AN Administrator 
I WANT TO be able to change existing user details such as display name and already existing roles.
SO THAT I can change roles or name

### Acceptance criteria

- [x] Personal identity code cannot be changed as there exists validation controls for it.
- [x] Notification when fields are empty with warning toast
- [x] Notification when form submission is successful
- [x] We do not check personal identity code validation in this task
- [x] We do not limit display name length or validate it somehow
- [x] Form adheres to branding

#### Integration

- [ ] Use endpoint `/cs-edit-user`
- [ ] Modal of "Muuda" / "Edit" on http://171.22.247.1:3001/haldus/kasutajad

**Comment by PaulaMerle** [comment]
Partly overlapping with #117

**Comment by janinakimtrohlev** [comment]
BUG - Personal identity code can be changed. It shouldn't be allowed. When changing name, title, e-mail under user details and saving, the account is deleted. Also it now allows to put some fields empty. But it should be notification when fields are empty with warning toast.

**Comment by ffrose** [comment]
BUG - When changing existing user details, personal identification code(Isikukood) field is missing and (Isikukood) value is shown in the "Ees- ja perekonnanimi" field. It should show first name and last name in "Ees- ja perekonnanimi" and (Isikukood) field and value should be hidden.

![Image](https://github.com/buerokratt/Buerokratt-Chatbot/assets/119657383/380ba362-f9b6-4ad7-be98-112815252346)



![Image](https://github.com/buerokratt/Buerokratt-Chatbot/assets/119657383/cfe4001f-ba20-4f78-ad0b-822727e527eb)

**Comment by 1AhmedYasser** [comment]
> 

@ffrose Noted, Will Fix it to show full name instead of id code and keep the id code field and value hidden when editing the user details

**Comment by 1AhmedYasser** [comment]
> BUG - When changing existing user details, personal identification code(Isikukood) field is missing and (Isikukood) value is shown in the "Ees- ja perekonnanimi" field. It should show first name and last name in "Ees- ja perekonnanimi" and (Isikukood) field and value should be hidden.
> 
> ![Image](https://user-images.githubusercontent.com/119657383/254225671-380ba362-f9b6-4ad7-be98-112815252346.png)
> 
> ![Image](https://user-images.githubusercontent.com/119657383/254225750-cfe4001f-ba20-4f78-ad0b-822727e527eb.png)

@ffrose Noted, Will Fix it to show full name instead of id code and keep the id code field and value hidden when editing the user details

**Comment by janinakimtrohlev** [comment]
> > BUG - When changing existing user details, personal identification code(Isikukood) field is missing and (Isikukood) value is shown in the "Ees- ja perekonnanimi" field. It should show first name and last name in "Ees- ja perekonnanimi" and (Isikukood) field and value should be hidden.
> > ![Image](https://user-images.githubusercontent.com/119657383/254225671-380ba362-f9b6-4ad7-be98-112815252346.png)
> > ![Image](https://user-images.githubusercontent.com/119657383/254225750-cfe4001f-ba20-4f78-ad0b-822727e527eb.png)
> 
> @ffrose Noted, Will Fix it to show full name instead of id code and keep the id code field and value hidden when editing the user details

Rgiht now i can change role and it is not deleting. But Name (nimi ja perekonnanimi) is not shown still. 
![image](https://github.com/buerokratt/Buerokratt-Chatbot/assets/122867977/fd053b71-eaf7-4803-b54f-dce79ad00d7b)

**Comment by 1AhmedYasser** [comment]
@janinakimtrohlev yes, this was due to being added in the first place without a full name due to a previous bug that is fixed at the moment, try editing the user and adding a name or add a new user to check it
